src/converters/groundTruth/sceneUpdateConverter.ts

```
@@ -326,69 +330,99 @@ export function convertGroundTruthToSceneUpdate(
-      timestamp,
-    ),
-    ...getDeletedEntities(
-      config.showReferenceLines ? osiGroundTruthReq.reference_line : [],
-      state.previousReferenceLineIds,
-      PREFIX_REFERENCE_LINE,
-      timestamp,
-    ),
-  ];
+  // Deletions logic (comparing previous step's entities with current step's entities).
+  //
+  // TODO(lichtblick-suite/lichtblick#1195): This per-consumer, data-driven
```

ESLint

[error] Unexpected 'todo' comment: 'TODO(lichtblick-suite/lichtblick#1195):...'.

(no-warning-comments)